src/cache/cache.go

```diff
@@ -88,7 +88,7 @@ func (c *Cache) ProcessTxEntry(newTx *types.Transaction, currentTime int64) (Pro
 
 	// no tx sent in the last d seconds
```

Comment:
You'd also need to add the logic here when a transaction entry is found with a nil value so that this transaction resends when the delay passes, the one removed by the commit below, please also check if you need to update the DB as this is a new transaction entry that is added: https://github.com/shutter-network/encrypting-rpc-server/pull/32/commits/a817af1caab7ad7cd4c1216b6bab91831d2f57b3#diff-5f62e51e1b548495647bedfc85fd09d507cdc34b67abdf4c4bbf5a01bc61afe7L68